charged-ieee/template/main.typ

```diff
@@ -37,9 +38,48 @@ To demonstrate the potential of Typst, we conducted a series of experiments comp
 
 Overall, we believe that Typst represents a significant step forward in the field of scientific writing and typesetting, providing researchers with a valuable tool to streamline their workflow and focus on what really matters: their research. In the following sections, we will introduce Typst in more detail and provide evidence for its superiority over other typesetting systems in a variety of scenarios.
 
-= Methods
-#lorem(90)
+= Methods <sec:methods>
+#lorem(45)
 
-$ a + b = gamma $
+$ a + b = gamma $<formula>
+
+#lorem(80)
+
+#figure(
+  placement: none,
```

Comment:
why do we `set figure(placement: top)` in the template, but then disable it everywhere?

Reply:
`placement: top` is the specified standard and makes sense especially with large figures and double-column figures.
Meanwhile `placement: none` is a personal preference for smaller figures that would otherwise be far from where they were referenced in the text. Feel free to get rid of this in `main.typ`

Reply:
I'm pretty sure bottom is allowed as well, no? We could use `placement: auto` if that makes more sense.

Reply:
> I'm pretty sure bottom is allowed as well, no? We could use `placement: auto` if that makes more sense.

Ah true, my bad.